src/v0/sources/shopify_v2/data/OrderCompletedConfig.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "sourceKeys": "checkout_id",
+    "destKey": "checkout_id",
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "sourceKeys": "id",
+    "destKey": "order_id",
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "sourceKeys": "subtotal_price",
+    "destKey": "subtotal",
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "sourceKeys": "total_price",
+    "destKey": "total",
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "sourceKeys": "total_discounts",
+    "destKey": "discount",
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "sourceKeys": {
+      "operation": "additionInArray",
+      "args": [
+        {
+          "propertyKey": "price",
+          "arrayKey": "shipping_lines"
+        }
+      ]
+    },
+    "destKey": "shipping",
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "sourceKeys": "total_tax",
+    "destKey": "tax",
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "sourceKeys": "total_discounts",
+    "destKey": "discount",
+    "metadata": {
+      "type": "toNumber"
+    }
+  },
+  {
+    "sourceKeys": {
+      "operation": "concatenationInArray",
+      "args": [
+        {
+          "propertyKey": "code",
+          "arrayKey": "discount_codes"
+        }
+      ]
```

Comment:
Can we concatenate with a delimiter support? Otherwise, it will be hard to split.

Reply:
comma delimiter (,) is used in concatenationInArray.